build.sbt

```diff
@@ -11,31 +11,45 @@ lazy val settings = Seq(
   )
 )
 
+lazy val commonAssemblySettings = Seq(
```

Comment:
What is the role of this libraries?

Reply:
It is stated in here.
https://github.com/sbt/sbt-assembly